main.go

```diff
@@ -20,6 +21,8 @@ func main() {
 		return
 	}
 
+	config.RecordingCallback = *recording
```

Comment:
**nit:** Can also use `flag.StringVar(&config.RecordingCallback, ...)` to avoid this step

Reply:
Nice